spyder_terminal/__init__.py

```diff
@@ -12,5 +12,5 @@
 if PYQT5:
     from .terminalplugin import TerminalPlugin as PLUGIN_CLASS
 
-VERSION_INFO = (0, 1, 1)
+VERSION_INFO = (0, 1, 2)
```

Comment:
The dev version should include a dev0, like this

https://github.com/spyder-ide/spyder/blob/master/spyder/__init__.py#L30